src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionPoolGroupProviderInfo.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if NETFRAMEWORK
+using System.Security;
+#endif
```

Comment:
#if-directive for namespaces is not required.
```suggestion
using System.Security;
```

Reply:
I put it there because only applies to PermissionSet which is only referenced by net framework

Reply:
I see. Namespaces usually don't hurt the execution and we could ignore them.